row/index.js

```diff
@@ -0,0 +1 @@
+export * from './src';
```

Comment:
I think this should be `dockit-columns` from a experience point of view

Reply:
yeap, I finally get it :D , because of the plural `columns` vs. `column` (which made no sense to me, but probably read it wrong before)